openapi/components/schemas/ReadyToPay/ReadyToPayMethods.yaml

```
@@ -21,12 +22,12 @@ example:
     feature: null
     brands:
       - Visa
-    bins:
-      mode: allow
-      values:
-        - "411111"
-        - "444433"
+    filters:
+      - "brand:Visa;bin:411111,444433"
   - method: ach
     feature: null
+    filters: []
   - method: paypal
+    filters: [ ]
```

Maybe you wanted to do next?
```suggestion
    filters: []
```
also to Skrill